common/source/docs/common-mp-tools.rst

```diff
@@ -33,12 +33,21 @@ This allows forwarding of your connected autopilots MAVLink stream via the inter
   :target: ../_images/mavlink-forwarding.jpg
 
 #. Select this tool
-#. Next a dialog will open with drop-downs to allow selection of communication port. For remote connections usually TCP will be chosen. For TCP or UDP, the baud rate selection box is unimportant.
+#. Next a dialog will open with drop-downs to allow selection of communication port. For remote connections usually UDP will be chosen. For TCP or UDP, the baud rate selection box is unimportant.
 #. Click this box if you want the remote party to be able to change parameters on you autopilot or control it remotely.
 #. When "Connect" is selected, depending on communications port, the public IP address of the recipient and port dialogs will open for input.
 
 After this, the remote site should start getting MAVLink packets from your autopilot. They can use whatever ground station they chose to receive them.
 
+.. note:: You can forward MAVLink packets to more than one location. Just enter a new connection, as above.
```

Comment:
i think it would replace it, not add another

Reply:
done